chef-basics/src/main/resources/logback.xml

```diff
@@ -20,7 +20,7 @@
 <configuration>
   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
     <encoder>
-      <pattern>-  %msg%n</pattern>
+      <pattern>%msg%n</pattern>
```

Comment:
I made this change to chef and compute, since those are the ones I'm familiar with. Each log in jclouds compute (and the compute/chef example code) is already prefixed with `>>` or `<<`, so it is pointless to have that `-` prefix. Logs look better without it :)